docs/sources/http.md

```diff
@@ -0,0 +1,52 @@
+# Event source for HTTP
```

Comment:
Nitpicky -I hate to do this one haha.  To follow the MLA guidelines for titles,` source` here should be capitalized as `Source `